AnnoyancesFilter/sections/mobile-app.txt

```diff
@@ -23,6 +23,7 @@
 ##head[prefix~="medium-com:"] ~ body .js-postActionsBarContent > .js-readNextInteractions
 !
 ||st.work.ua/js/app-banner-
+jp.pinkoi.com###g-app-bar
```

Comment:
Add `pinkoi.com#$#.s-app-banner div#m-mainview { padding-top: 51px!important; }` above or below this rule.

Reply:
done